pkg/controller/persistentvolume/persistentvolume_controller.go

```diff
@@ -73,6 +73,30 @@ const annBoundByController = "pv.kubernetes.io/bound-by-controller"
 // provisioning to choose a particular storage class (aka profile).
 const annClass = "volume.alpha.kubernetes.io/storage-class"
 
+// This annotation is added to a PV that has been dynamically provisioned by
+// Kubernetes. It's value is name of volume plugin that created the volume.
```

Comment:
Volume plugins are free to add another more user-friendly annotations. This is mostly for internal controller state, the value does not matter much (same as other controller annotations).